tests/test_pixel.py

```diff
@@ -1477,8 +1487,6 @@ def test_des2():
     # Basically saying the solution doesn't go completely haywire.
     np.testing.assert_allclose(test_stamp.array, check_stamp.array, rtol=tol_qrp, atol=tol_qrp)
 
-    # Also exercise the SVD fallback to the non-QR method.
-    config['psf']['interp']['use_qr'] = False
```

Comment:
Why did you remove this?

Reply:
I did not see it was switch to true earlier in the code... I added it back.